gulpfile.js

```diff
@@ -88,34 +88,35 @@ var paths = {
   },
 
   vendorChallenges: [
-    'public/bower_components/jshint/dist/jshint.js',
-    'public/bower_components/chai/chai.js',
-    'public/bower_components/CodeMirror/lib/codemirror.js',
-    'public/bower_components/CodeMirror/addon/comment/comment.js',
-    'public/bower_components/CodeMirror/addon/edit/closebrackets.js',
-    'public/bower_components/CodeMirror/addon/edit/matchbrackets.js',
-    'public/bower_components/CodeMirror/addon/lint/lint.js',
-    'public/bower_components/CodeMirror/addon/lint/javascript-lint.js',
-    'public/bower_components/CodeMirror/mode/javascript/javascript.js',
-    'public/bower_components/CodeMirror/mode/xml/xml.js',
-    'public/bower_components/CodeMirror/mode/css/css.js',
-    'public/bower_components/CodeMirror/mode/htmlmixed/htmlmixed.js',
+    'node_modules/jshint/dist/jshint.js',
+    'node_modules/chai/chai.js',
+    'node_modules/codemirror/lib/codemirror.js',
+    'node_modules/codemirror/addon/comment/comment.js',
+    'node_modules/codemirror/addon/edit/closebrackets.js',
+    'node_modules/codemirror/addon/edit/matchbrackets.js',
+    'node_modules/codemirror/addon/lint/lint.js',
+    'node_modules/codemirror/addon/lint/javascript-lint.js',
+    'node_modules/codemirror/mode/javascript/javascript.js',
+    'node_modules/codemirror/mode/xml/xml.js',
+    'node_modules/codemirror/mode/css/css.js',
+    'node_modules/codemirror/mode/htmlmixed/htmlmixed.js',
```

Comment:
This is still a problem. I don't think shrinkwrap is a viable option. You could try using the path module to programmatically determine the routes to projects.